qdrant-landing/content/documentation/hybrid-cloud/hybrid-cloud-cluster-creation.md

```diff
@@ -0,0 +1,158 @@
+
+You can expose your Qdrant clusters to your client applications using Kubernetes services and ingresses. By default, a `ClusterIP` service is created for each Qdrant cluster.
+
+Within your Kubernetes cluster, you can access the Qdrant cluster using the service name and port:
+
+```
+http://qdrant-9a9f48c7-bb90-4fb2-816f-418a46a74b24.qdrant-namespace.svc:6333
+```
+
+This endpoint is also visible on the cluster detail page.
+
+If you want to access the database from your local developer machine, you can use `kubectl port-forward` to forward the service port to your local machine:
+
+```
+kubectl --namespace your-qdrant-namespace port-forward service/qdrant-9a9f48c7-bb90-4fb2-816f-418a46a74b24 6333:6333
+```
+
+You can also expose the database outside the Kubernetes cluster with a `LoadBalancer` (if supported in your Kubernetes environment) or `NodePort` service or an ingress.
+
+The service type and necessary annotations can be configured in the "Kubernetes Configuration" section during cluster creation, or on the cluster detail page.
+
+Especially if you create a LoadBalancer Service, you may need to provider annotations for the loadbalancer configration. Please refer to the documention of your cloud provider for more details.
+
+Examples:
+
+* [AWS EKS LoadBalancer annotations](https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/guide/ingress/annotations/)
+* [Azure AKS Public LoadBalancer annotations](https://learn.microsoft.com/en-us/azure/aks/load-balancer-standard)
+* [Azure AKS Internal LoadBalancer annotations](https://learn.microsoft.com/en-us/azure/aks/internal-lb)
+* [GCP GKE LoadBalancer annotations](https://cloud.google.com/kubernetes-engine/docs/concepts/service-load-balancer-parameters)
+
+You could also create a Loadbalancer service manually like this:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: qdrant-9a9f48c7-bb90-4fb2-816f-418a46a74b24-lb
+  namespace: qdrant-namespace
+spec:
+  type: LoadBalancer
+  ports:
+  - name: http
+    port: 6333
+  - name: grpc
+    port: 6334
+  selector:
+    app: qdrant
+    cluster-id: 9a9f48c7-bb90-4fb2-816f-418a46a74b24
+```
+
+An ingress could look like this:
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: qdrant-9a9f48c7-bb90-4fb2-816f-418a46a74b24
+  namespace: qdrant-namespace
+spec:
+    rules:
+    - host: qdrant-9a9f48c7-bb90-4fb2-816f-418a46a74b24.your-domain.com
+      http:
+        paths:
+        - path: /
+          pathType: Prefix
+          backend:
+            service:
+              name: qdrant-9a9f48c7-bb90-4fb2-816f-418a46a74b24
+              port:
+                number: 6333
+```
+
+Please refer to the Kubernetes, ingress controller and cloud provider documention for more details.
+
+If you expose the database with such a way, you will be able to see this also reflected as an endpoint on the cluster detail page. And will see the Qdrant database dashboard link pointing to it.
```

Comment:
'With such a way' doesn't make too much sense 

'In such a way' is better

'Like this' is more concise and better still